### ISSUE TITLE
Link component does not work with keyboard if only click event is bound to it

### ISSUE THREAD

**Issue by jjlankinen** [otavanopisto/muikku#5134]
When we use `<Link>`component to open dialog, act as a button or trigger dropdown submenu there is severe problems for keyboard users currently. 

Rendered `<Link>` does not trigger the onclick event when using only keyboard. Enter and spacebar (space should activate the link only if the <Link> is rendered as an actual html button) should trigger said onclick event. 

If link has `href` attribute also set then keyboard can be used to trigger the action.
